src/Components/List/List.jsx

```diff
@@ -2,21 +2,54 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { db } from '../../lib/firebase.js';
 import { formatString } from '../../lib/helpers.js';
+import calculateEstimate from '../../lib/estimates';
 import dayjs from 'dayjs';
 import './List.css';
 
+const getNumberOfPurchases = (item) => {
+  if (item.numberOfPurchases === undefined) {
+    return 1;
+  } else {
+    return item.numberOfPurchases + 1;
+  }
+};
+
+const getLastPurchaseDate = (item, currentDate) => {
+  if (item.lastPurchased === null) {
+    return currentDate;
+  } else {
+    dayjs(item.lastPurchased.toDate());
```

Comment:
Is this line missing a `return`?

Reply:
It worked without the return but we added it for readability. Thanks!